src/test/java/com/synopsys/integration/stepworkflow/StepWorkflowTest.java

```diff
@@ -17,26 +22,62 @@
     public static final Integer TEST_DATA_FIVE = 5;
 
-    private static final String exceptionMessage1 = "Exception #1";
-    private static final IntegrationException exception = new IntegrationException(exceptionMessage1);
+    private static final IntegrationException integrationException = new IntegrationException("Exception #1");
 
-    private final SubStep<Object, Integer> successfulSupplierOne = SubStep.ofSupplier(this::successfulDataSupplierOne);
-    private final SubStep<Object, Integer> successfulSupplierTwo = SubStep.ofSupplier(this::successfulDataSupplierTwo);
-    private final SubStep<Object, Integer> successfulSupplierThree = SubStep.ofSupplier(this::successfulDataSupplierThree);
-    private final SubStep<Object, Integer> unsuccessfulSupplierOne = SubStep.ofSupplier(this::unsuccessfulDataSupplier);
+    private SubStep<Object, Integer> subStepSupplierOne;
+    private SubStep<Object, Integer> subStepSupplierTwo;
+    private SubStep<Object, Integer> subStepSupplierThree;
+    private SubStep<Object, Object> subStepExecutorOne;
+    private SubStep<Integer, Object> subStepFunctionOne;
+    private SubStep<Integer, Object> subStepFunctionTwo;
+    private StepWorkflow.FlowController<Object, Integer> flowControllerOne;
+    private StepWorkflow.Builder<Integer> successfulBuilderOfSupplierOne;
+    private StepWorkflow.Builder<Integer> builderThenSubStepSupplierTwo;
 
-    private final SubStep<Object, Object> successfulExecutorOne = SubStep.ofExecutor(this::successfulExecutor);
-    private final SubStep<Integer, Object> successfulFunctionOne = SubStep.ofFunction(this::successfulFunctionOne);
-    private final SubStep<Integer, Object> successfulFunctionTwo = SubStep.ofFunction(this::successfulFunctionTwo);
+    private StepWorkflow.Builder<Integer> unsuccessfulBuilderOfSupplierOne;
+    private SubStep<Object, Integer> unsuccessfulSubStepSupplierOne;
 
-    private final StepWorkflow.FlowController<Object, Integer> successfulFlowControllerOfSupplier = new StepWorkflow.FlowController<>(successfulSupplierOne);
+    private static Stream<Boolean> runConditionalWorkflowObjectToTest() {
+        return Stream.of(true, false);
+    }
 
-    private final StepWorkflow.Builder<Integer> successfulBuilderOfSupplierOne = StepWorkflow.first(successfulSupplierOne);
-    private final StepWorkflow.Builder<Integer> unsuccessfulBuilderOfSupplierOne = StepWorkflow.first(unsuccessfulSupplierOne);
+    @BeforeEach
+    public void setup() {
+        subStepSupplierOne = SubStep.ofSupplier(this::successfulDataSupplierOne);
+        subStepSupplierTwo = SubStep.ofSupplier(this::successfulDataSupplierTwo);
+        subStepSupplierThree = SubStep.ofSupplier(this::successfulDataSupplierThree);
+        subStepExecutorOne = SubStep.ofExecutor(this::successfulExecutor);
+        subStepFunctionOne = SubStep.ofFunction(this::successfulFunctionOne);
+        subStepFunctionTwo = SubStep.ofFunction(this::successfulFunctionTwo);
```

Comment:
The state of these guys never change, we should just initialize them as final variables on lines 27 through 32.